File: OCaml/FizzBuzz/fizzbuzz.ml
let fizzbuzz i =
  match i mod 3, i mod 5 with
    0, 0 -> "FizzBuzz"
  | 0, _ -> "Fizz"
  | _, 0 -> "Buzz"
  | _    -> string_of_int i;;

let run n =
  for i = 1 to n do print_endline (fizzbuzz i) done;;

run 15;;

File: CSharp/FizzBuzz/fizzbuzz.cs
using System;

public class FizzBuzz {
    public static void Main() {
        run(15);
    }

    private static void run(int n) {
        for(int i = 1; i <= n; i++) {
            if (i % 15 == 0) {
                Console.WriteLine("FizzBuzz");
            } else if (i % 3 == 0) {
                Console.WriteLine("Fizz");
            } else if (i % 5 == 0) {
                Console.WriteLine("Buzz");
            } else {
                Console.WriteLine(i);
            }
        }
    }
}
